databases/metadata/src/main/conversions/c210_2015081002.clj

```diff
@@ -0,0 +1,24 @@
+(ns facepalm.c210-2015081002
```

Comment:
I left a copy of this comment on the commit itself. Sorry for the duplication. `src/main/data/99_version.sql` should also be updated to include the latest version number. This helps to prevent conversions from being done unnecessarily.